samples/client/petstore/rust/README.md

```diff
@@ -0,0 +1,97 @@
+# Go API client for swagger
```

Comment:
Yes, we usually run the shell script (./bin/) or windows batch file (.\bin\windows\) to regenerate the Petstore samples so that CI (travis, circleci, etc) can verify the change (there's no CI setup for Rust Petstore sample yet)